### PR TITLE
[bug 1254981] Replace cache_page with cache_control.

### PR DESCRIPTION
**Pull request by johngian**
* Instead of using CACHE backend to cache response in the
  view level, add `cache-control` headers.

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: mozillians/phonebook/views.py [hunk 1]
```diff
@@ -431,7 +431,7 @@ def betasearch(request):
 
 
 @allow_public
-@cache_page(60 * 60 * 168)  # 1 week.
+@cache_control(must_revalidate=True, max_age=60 * 60 * 168)  # 1 week.
```
Comment (akatsoulas) [inline mozillians/phonebook/views.py]: I would suggest to rewrite the `max_age` as `3600 * 24 *7`

Comment (akatsoulas) [inline mozillians/phonebook/views.py]: Maybe you can also add `public=True`

**Comment by akatsoulas** [comment]
r+wc :coffee:

**Pull request merged**